src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -15,13 +15,15 @@ interface PropsType {
 const MAX_AVATARS_VISIBLE = 2;
 
 export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
-  const partygoers = usePartygoers();
+  const { recentWorldUsers } = useRecentWorldUsers();
+
+  // @debt Separete room attendies into a separate hook
   const usersInRoom = useMemo(
     () =>
-      partygoers?.filter(
+      recentWorldUsers.filter(
         (partygoer) => partygoer.lastSeenIn?.[`${venue.name}/${room.title}`]
       ),
-    [partygoers, venue.name, room.title]
+    [recentWorldUsers, venue.name, room.title]
   );
```

Comment:
This would be resolved by implementing `useRecentVenueUsers` as described elsewhere in this review.